pkg/identity/certificated.go

```diff
@@ -82,6 +82,42 @@ func Certificated(idConfig *IdentityConfig, stopChan <-chan struct{}) error {
 		return w.Save()
 	}
 
+	identityProvisioningRequest := func(idConfig *IdentityConfig, handler *identityHandler, forceInit bool) (err error, identity *InstanceIdentity, keyPem []byte) {
```

Comment:
@ctyano I was wondering if we should have a log message with `forceInit` value to differentiate between normal and unusual call of the function`identityProvisioningRequest`

`if forceInit { ... message ... } ` would be enough? Would like to hear your opinion.

Reply:
was handled here:
https://github.com/AthenZ/k8s-athenz-sia/pull/12/commits/4f07ea168903882a57a509b2e8439e3fb5432976